packages/system-upgrade-controller/generated-changes/patch/Chart.yaml.patch

```diff
@@ -12,8 +12,8 @@
  sources:
 -  - "https://github.com/rancher/system-charts/blob/dev-v2.6/charts/rancher-k3s-upgrader"
 +  - "https://github.com/rancher/system-charts/charts/system-upgrade-controller"
```

Comment:
We should drop those 2 lines to keep the source URL unchanged. 

Also, change the above home URL to `https://github.com/rancher/system-charts/blob/dev-v2.6/charts/rancher-k3s-upgrader`

Reply:
I believe this means that we will leave the source and home URLs unchanged from the "upstream" chart. I will update this.